projects/popovers/src/public-api.ts

```diff
@@ -0,0 +1,25 @@
+export * from './modules/dropdown/types/dropdown-horizontal-alignment';
+export * from './modules/dropdown/types/dropdown-menu-change';
+export * from './modules/dropdown/types/dropdown-message';
+export * from './modules/dropdown/types/dropdown-message-type';
+export * from './modules/dropdown/types/dropdown-trigger-type';
+export * from './modules/dropdown/dropdown.module';
+
+export * from './modules/popover/popover.module';
+export * from './modules/popover/types/popover-alignment';
+export * from './modules/popover/types/popover-message';
+export * from './modules/popover/types/popover-message-type';
+export * from './modules/popover/types/popover-placement';
+export * from './modules/popover/types/popover-position';
+export * from './modules/popover/types/popover-trigger';
+
+// Components and directives must be exported to support Angular's "partial" Ivy compiler.
+// Obscure names are used to indicate types are not part of the public API.
+export { SkyDropdownItemComponent as λ1 } from './modules/dropdown/dropdown-item.component';
+export { SkyDropdownButtonComponent as λ2 } from './modules/dropdown/dropdown-button.component';
+export { SkyDropdownComponent as λ3 } from './modules/dropdown/dropdown.component';
+export { SkyDropdownComponent as λ4 } from './modules/dropdown/dropdown.component';
```

Comment:
You got a duplicate export, here, which is causing the strange build error.

Reply:
fixed in last commit